pgjdbc/src/test/java/org/postgresql/test/jdbc4/ArrayTest.java

```diff
@@ -53,6 +56,10 @@ public void setUp() throws Exception {
     super.setUp();
     _conn = con;
 
+    Properties props = new Properties();
+    PGProperty.STRING_TYPE.set(props, "unspecified");
+    _stringTypeUnspecifiedConn = TestUtil.openDB(props);
```

Comment:
Please move that to its own test class and use `BaseTest4.updateProperties`: https://github.com/pgjdbc/pgjdbc/blob/8be516d47ece60b7aeba5a9474b5cac1d538a04a/pgjdbc/src/test/java/org/postgresql/test/jdbc2/BaseTest4.java#L35